### PR TITLE
humanize time delta into a proper format rather than 48964518 sec.

### PR DESCRIPTION
**Pull request by PatWie**

### CODE REVIEW

**Comment by ppwwyyxx** [comment]
Can this be done by just using standard python library rather than a 3rd-party one?

**Comment by PatWie** [comment]
sure see edits

File: tensorpack/callbacks/group.py [hunk 1]
```diff
@@ -30,13 +31,14 @@ def timed_callback(self, name):
...
     def log(self):
+
         """ log the time of some heavy callbacks """
         if self.tot < 3:
             return
         msgs = []
         for name, t in self.times:
             if t / self.tot > 0.3 and t > 1:
-                msgs.append("{}: {:.3f}sec".format(name, t))
+                msgs.append(humanize_time_delta(t))
```
Comment (ppwwyyxx) [inline tensorpack/callbacks/group.py]: name was missing

File: tensorpack/utils/utils.py [hunk 2]
```diff
@@ -16,9 +16,63 @@
...
+    units = ['week', 'day', 'hour', 'minute', 'second']
+    vals = [
+        isec / 60 / 60 / 24 / 7,
+        (isec / 60 / 60 / 24) % 7,
+        (isec / 60 / 60) % 24,
+        (isec / 60) % 60,
+        isec % 60
+    ]
+    if sec < 60:
+        vals[-1] = sec
```
[the remark this answers is not in the document]
Comment (ppwwyyxx) [reply tensorpack/utils/utils.py]: An easier way would be like: `datetime.datetime(2000,1,1) + datetime.timedelta(seconds=9999999)`

**Pull request merged**